faust/transport/drivers/aiokafka.py

```diff
@@ -840,14 +840,14 @@ def _verify_aiokafka_event_path(self, now: float, tp: TP) -> bool:
         poll_at = None
         aiotp_state = assignment.state_value(aiotp)
         if aiotp_state and aiotp_state.timestamp:
-            poll_at = aiotp_state.timestamp / 1000
```

Comment:
Looks like this change was originally implemented in https://github.com/faust-streaming/faust/commit/dcc421fc29a8b555ff38a0bfac4366fcc04d2369 by @patkivikram, could you expand on this?